### ISSUE TITLE
Question about KPlanes result of DyNeRF dataset?

### ISSUE THREAD

**Issue by xuyaojian123** [zju3dv/4K4D#18]
![image](https://github.com/zju3dv/4K4D/assets/51220835/e2b1f798-7f25-4125-99dc-bfd03b17627e)

May I ask whether your k-planes results are from orginal paper results or from your own actual runs to get? I trained the K-planes hybrid version using the DyNeRF dataset, following K-planes config file without making any other changes to the configuration.  There are difference between the results in the paper and my results. https://github.com/sarafridov/K-Planes/issues/30#issue-1798836479

**Comment by dendenxu** [comment]
Hi @xuyaojian123 , we used their original code without change. But since our setting is different from theirs (we used the whole 300 frames sequence and their configs only support 50) , we modified the config to train on all 300 frames.

On the note of different metrics of yours than their paper, I must say it could get very hard to have an exact match of the numbers since different machines (gpus) use different heuristics in their underlying algorithms. In my experience, 3090 usually reports more stable results than 4090, and 4090 than A100 (so all of our quantitative experiments are done on 3090s). And if you trained the model on one machine (for example, a 3090), running the evaluation on a 4090 will produce a different metric number (usually worse). So maybe you could (1) rerun their experiments on a 3090 or (2) ask them for qualitative comparison images.

**Comment by xuyaojian123** [comment]
谢谢，感谢你的回答，我也是在3090跑的，主要是我的结果和论文结果差距很大，特别是ssim指标。我需要在他的基础上做模型大小压缩。请问你在k-planes代码中修改了哪里让他训练全部300帧，而不是50帧，从而结果和他论文差不多的结果，感谢！！！

**Comment by xuyaojian123** [comment]
![9d07b23b16a5e5a546665686d84994e](https://github.com/zju3dv/4K4D/assets/51220835/713dd7aa-87c9-47dd-84bf-8e7a9b15697f)

![image](https://github.com/zju3dv/4K4D/assets/51220835/b48473ca-555c-4786-93f0-e2e9dcf85ae8)

Maybe code is modified  here， set load_every to 1. In this way, all frames in the training set can be used? I'll run the code later.

**Comment by dendenxu** [comment]
> 谢谢，感谢你的回答，我也是在3090跑的，主要是我的结果和论文结果差距很大，特别是ssim指标。我需要在他的基础上做模型大小压缩。请问你在k-planes代码中修改了哪里让他训练全部300帧，而不是50帧，从而结果和他论文差不多的结果，感谢！！！

Sorry，我发现我看错你说的baseline了🤦‍，我们改的是hyperreel的配置，不是kplanes的。

**Comment by dendenxu** [comment]
我们是直接跑的原版代码，看起来是你的evaluation方式没有和他们的code完全align？

**Comment by xuyaojian123** [comment]
> 我们是直接跑的原版代码，看起来是你的evaluation方式没有和他们的code完全align？

没有吧，你可以发其中任意一个场景你训练好的模型给我嘛，我想看看你的结果是和它一样的嘛？或者能不能添加一个联系方式 vx:18679338489，私聊一下（等会这个会删除了）
